ngraph/python/tests/runtime.py

```diff
@@ -111,6 +134,7 @@ def __call__(self, *input_values: NumericData) -> List[NumericData]:
             )
         for parameter, input in zip(self.parameters, input_values):
             parameter_shape = parameter.get_output_partial_shape(0)
+            input = input.reshape(add_batch_size_to_shape_workaround(input.shape, parameter_shape))
```

Comment:
Can we use [set_batch](https://github.com/openvinotoolkit/openvino/blob/2be6032c8dc79d420b8ad52e73afe40d5f507a99/inference-engine/ie_bridges/python/src/openvino/inference_engine/ie_api.pyx#L1257) on executable_network instead of this workaround? (more details in https://github.com/openvinotoolkit/openvino/blob/master/docs/IE_DG/ShapeInference.md)

Reply:
Shapes are fine in the network. Problem is with the input/output data (*.pb files).

Reply:
ok, I see it now ;)